docs/service/connections.md

```diff
@@ -0,0 +1,241 @@
+# Connection Service
+
+The Connection Service provides RESTful APIs for managing the lifecycle of
+connections. All connection information is stored in the connections
+store of the dataset.
+
+These are the lifecycle operations supported by the connection service:
+
+* Create a new connection (**POST**, ```${base}/connections/create```)
+* Update an entire connection (**POST**, ```${base}/connections/{id}/update```)
+* Update properties of a connection (**PUT**, ```${base}/connections/{id}/properties?key=<key>&value=<value>```)
+* Retrieve all properties of a connection (**GET**, ```${base}/connections/{id}/properties```)
+* Delete a connection (**DELETE**, ```${base}/connections/{id}```)
+* Clone a connection (**GET**, ```${base}/connections/{id}/clone```)
+* Retrieve information about all of the connections (**GET**, ```${base}/connections```)
+* Retrieve information about a connection (**GET** ```${base}/connections/{id}```)
+
+## Base
+
+This is the base URL for the service:
+
+```
+http://localhost:11015/v3/namespaces/default/apps/dataprep/services/service/methods
```

Comment:
this is technically the namespace where dataprep is deployed. Should we put a placeholder for namespace?